Rover.podspec

```diff
@@ -43,29 +43,34 @@ Pod::Spec.new do |s|
     ss.source_files = "Sources/Location/**/*.swift"
     ss.resources = "Sources/Location/Model/RoverLocation.xcdatamodeld"
     ss.dependency "Rover/Data"
```

Comment:
RoverData is a transient dependency of RoverUI so you can remove this dependency if the subspec depends on RoverUI.

Reply:
Same for all other subspacs that depend on RoverUI now.

Reply:
gonna review your review! "transitive" ;)